.github/workflows/test.yml

```diff
@@ -0,0 +1,33 @@
+name: test
+on: push
```

Comment:
Oh wait, this will only run on branch pushes. We should also add `pull_request` for this to be a proper commit check, right @chadwhitacre?

Reply:
Yup, good call. Maybe https://github.com/getsentry/onpremise/pull/639 does what we want?